src/qgis_stac/ui/connection_dialog.ui

```diff
@@ -0,0 +1,419 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There is an issue with the connection dialog UI, some buttons for toggling tabs is overlapped in a wrong place see
![connection_dialog](https://user-images.githubusercontent.com/2663775/142836323-0f6d0e09-a716-4784-a1d2-2e159a0dc4fe.png)